samples/excel/85-preview-apis/data-types-formatted-number.yaml

```
@@ -0,0 +1,149 @@
+              [
+                {
+                  type: Excel.CellValueType.formattedNumber,
+                  basicValue: 12.34,
+                  numberFormat: "$* #,##0.00"
+                }
+              ]
+            ];
+
+            await context.sync();
+          });
+        }
+
+        async function getFormattedNumber() {
+          await Excel.run(async (context) => {
+            // This method prints information about data types 
+            // in cells A1 and A2 to the console.
+
+            // Get the active worksheet and two ranges on that sheet.
+            const sheet = context.workbook.worksheets.getActiveWorksheet();
+            const currencyRange = sheet.getRange("A2");
+            const dateRange = sheet.getRange("A1");
+
+            // Load the property of the ranges that contains data types.
+            currencyRange.load(["valuesAsJson"]);
+            dateRange.load(["valuesAsJson"]);
+            await context.sync();
+
+            // Access the data types property.
+            const currencyValues = currencyRange.valuesAsJson[0][0];
+            const dateValues = dateRange.valuesAsJson[0][0];
+
+            // Print information about the data types to the console.
+            console.log("Date");
+            console.log("  Type: " + dateValues.type);
+            console.log("  Basic value: " + dateValues.basicValue);
+            console.log("  Basic type: " + dateValues.basicType);
+            console.log("  Number format: " + dateValues.numberFormat);
+
+            console.log("Currency");
+            console.log("  Type: " + currencyValues.type);
+            console.log("  Basic value: " + currencyValues.basicValue);
+            console.log("  Basic type: " + currencyValues.basicType);
+            console.log("  Number format: " + currencyValues.numberFormat);
+          });
+        }
+
+        /** Default helper for invoking an action and handling errors. */ async function tryCatch(callback) {
+          try {
+            await callback();
+          } catch (error) {
+            // Note: In a production add-in, you'd want to notify the user through your add-in's UI.
+            console.error(error);
+          }
+        }
+    language: typescript
+template:
+    content: |-
+        <section class="ms-font-m">
+          <p>This sample shows how to work with formatted number data types. The sample sets a number formatted as a date,
+            and a number formatted as currency. It also prints information about these data types to the console.</p>
+        </section>
+        <section class="setup ms-font-m">
+          <h3>Try it out</h3>
+          <button id="set-formatted-number-date" class="ms-Button">
+              <span class="ms-Button-label">Set formatted date number</span>
+          </button>
+          <button id="set-formatted-number-currency" class="ms-Button">
+              <span class="ms-Button-label">Set formatted currency number</span>
```

Consider a shorter button name so it doesn't run the risk of wrapping.
```suggestion
              <span class="ms-Button-label">Set formatted currency</span>
```